README-spacetime.md

```diff
@@ -0,0 +1,75 @@
+# Spacetime profiling
+
+Spacetime is a feature for showing memory usage over time within OCaml programs. It's useful
+for revealing the location of memory leaks.
+
+## Building for Spacetime
+
+To use Spacetime, you need an OCaml compiler built with that feature enabled. There are
+Spacetime-enabled compiler switches within OPAM. For example, there's the switch 
+4.07.1+spacetime, which works with same OPAM packages as vanilla 4.07.1. To build
+with it in your enviroment, install the switch:
+
+```
+  opam switch create 4.07.1+spacetime # or the current OCaml version for building Coda 
+  eval $(opam env)
+```
+
+Then follow the instructions for "Building outside docker" given in README-dev.md. Building
+some OPAM packages with Spacetime requires a large amount of stack space, so you
```

Comment:
Can we pop the limit comment above the install instructions? Might be missed if someone is following the instructions sequentially, which could lead to some frustration!